facility-manager.html

```diff
@@ -528,6 +541,61 @@
     <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+<div class="line-block">
+    <div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="lightadmin"></a>Creating administrators with restricted privileges</h2>
+
+<p>This requires using the OMERO.web Admin tool.</p>
+
+<p>Both administrators with restricted privileges (restricted administrators) and full
+   administrators can create restricted administrators. The privileges of the new
```

Comment:
system administrators(administrators with full privileges) instead of full administrators ?